### PR TITLE
[BZ-1790551]  Add pointer to etcd storage requirements

### PR DESCRIPTION
**Pull request by tmalove**
Preview link: http://file.rdu.redhat.com/tlove/bz-etcd-1790551/post_installation_configuration/storage-configuration.html#other-specific-application-storage-recommendations

Fixes: Bug https://bugzilla.redhat.com/show_bug.cgi?id=1790551
Urgency: Medium
Applies to: 4.6 +

### CODE REVIEW

**Comment by tmalove** [comment]
@oarribas @mffiedler  Review the update in the 'Other specific application storage recommendations' topic, per [BZ 1790551](https://bugzilla.redhat.com/show_bug.cgi?id=1790551)

Thanks!

**Comment by tmalove** [comment]
@oarribas I corrected the link and included the reference to sufficient disk performance. About the URL containing a specific release, I am looking into this and will have an answer for you on that one in the next few days.
Here is the preview link:

https://deploy-preview-43626--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/optimizing-storage.html#other-specific-application-storage-recommendations

**Comment by tmalove** [comment]
@geliu2016  review request: please verify the update in this module (last bullet). Thanks!

**Review by geliu2016 — APPROVED** [review]
/lgtm

File: modules/recommended-configurable-storage-technology.adoc [hunk 1]
```diff
@@ -124,3 +124,4 @@ Application use cases vary from application to application, as described in the
 * It is highly recommended that you use `etcd` with storage that handles serial writes (fsync) quickly, such as NVMe or SSD. Ceph, NFS, and spinning disks are not recommended.
 * {rh-openstack-first} Cinder: {rh-openstack} Cinder tends to be adept in ROX access mode use cases.
 * Databases: Databases (RDBMSs, NoSQL DBs, etc.) tend to perform best with dedicated block storage.
+* The etcd database must have enough storage and adequate performance capacity to enable a large cluster. For more information, see https://docs.openshift.com/container-platform/4.10/scalability_and_performance/recommended-host-practices.html#recommended-etcd-practices_recommended-host-practices[Recommended etcd practices].
```
Comment (mburke5678) [inline modules/recommended-configurable-storage-technology.adoc:127]: @tmalove Are we now allowed to add links or hard links to modules?

**Review by mffiedler — APPROVED** [review]
/lgtm

**Comment by tmalove** [comment]
@oarribas @geliu2016  I moved the link outside of the module and into an 'Additional resources' section adjacent to the module. The content is basically the same however, I request that you take a look at the updated format because it the format has been changed. Thanks!

Updated preview link: http://file.rdu.redhat.com/tlove/bz-etcd-1790551/post_installation_configuration/storage-configuration.html#other-specific-application-storage-recommendations

File: post_installation_configuration/storage-configuration.adoc [hunk 2]
```diff
@@ -69,6 +69,12 @@ include::modules/available-persistent-storage-options.adoc[leveloffset=+1]
 
 include::modules/recommended-configurable-storage-technology.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+xref:../scalability_and_performance/recommended-host-practices.adoc#recommended-etcd-practices_recommended-host-practices[Recommended etcd practices]
```
Comment (bergerhoffer) [inline post_installation_configuration/storage-configuration.adoc:75]: This should be a bullet

**Review by bergerhoffer — COMMENTED** [review]
Just one last thing, otherwise this looks great!

**Pull request merged**